e2e_tests/tests/conftest.py

```diff
@@ -81,6 +81,13 @@ def pytest_addoption(parser: Parser) -> None:
     parser.addoption("--follow-local-logs", action="store_true", help="Follow local docker logs")
 
 
+@pytest.fixture(scope="session", autouse=True)
+def instantiate_gpu() -> None:
+    command = ["det", "cmd", "--config", "resources.slots=1", "'sleep 30'"]
```

Comment:
should save a tiny bit of time
```suggestion
    command = ["det", "cmd", "--config", "resources.slots=1", "'sleep 1'"]
```